plots/laws/relativistic/relativistic_force_for_different_acceleration_configurations.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+from collections import namedtuple
+from sympy import Symbol, plot
+from sympy.physics.units import speed_of_light
+from symplyphysics import Vector, vector_magnitude
+from symplyphysics.laws.relativistic.vector import force_via_acceleration
+
+Datum = namedtuple("Datum", "label acceleration")
+
+data_ = [
+    Datum(
+        label=r"$a_{\perp} = 0$",
+        acceleration=Vector([10, 0, 0]),
+    ),
+    Datum(
+        label=r"$a_{\parallel} = 0$",
+        acceleration=Vector([0, 10, -20]),
+    ),
+    Datum(
+        label=r"$a_{\perp} \ne 0, a_{\parallel} \ne 0$",
+        acceleration=Vector([10, 10, -20]),
```

Comment:
I suppose these vectors should have the same magnitude, to make this plot somehow informative.

Reply:
This makes sense. I'll do that